2024-12-04 16:00:32,419 INFO    MainThread:2404540 [wandb_setup.py:_flush():79] Current SDK version is 0.18.7
2024-12-04 16:00:32,419 INFO    MainThread:2404540 [wandb_setup.py:_flush():79] Configure stats pid to 2404540
2024-12-04 16:00:32,419 INFO    MainThread:2404540 [wandb_setup.py:_flush():79] Loading settings from /home/woocheol/.config/wandb/settings
2024-12-04 16:00:32,419 INFO    MainThread:2404540 [wandb_setup.py:_flush():79] Loading settings from /home/woocheol/LNS-PBS-representation-new/wandb/settings
2024-12-04 16:00:32,419 INFO    MainThread:2404540 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-12-04 16:00:32,419 INFO    MainThread:2404540 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-12-04 16:00:32,419 INFO    MainThread:2404540 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program_relpath': 'main_nn.py', 'program_abspath': '/home/woocheol/LNS-PBS-representation-new/main_nn.py', 'program': '/home/woocheol/LNS-PBS-representation-new/main_nn.py'}
2024-12-04 16:00:32,419 INFO    MainThread:2404540 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-12-04 16:00:32,420 INFO    MainThread:2404540 [wandb_setup.py:_flush():79] Applying login settings: {'api_key': '***REDACTED***'}
2024-12-04 16:00:32,420 INFO    MainThread:2404540 [wandb_init.py:_log_setup():533] Logging user logs to /home/woocheol/LNS-PBS-representation-new/wandb/run-20241204_160032-l8vc757h/logs/debug.log
2024-12-04 16:00:32,420 INFO    MainThread:2404540 [wandb_init.py:_log_setup():534] Logging internal logs to /home/woocheol/LNS-PBS-representation-new/wandb/run-20241204_160032-l8vc757h/logs/debug-internal.log
2024-12-04 16:00:32,420 INFO    MainThread:2404540 [wandb_init.py:init():619] calling init triggers
2024-12-04 16:00:32,420 INFO    MainThread:2404540 [wandb_init.py:init():626] wandb.init called with sweep_config: {}
config: {}
2024-12-04 16:00:32,420 INFO    MainThread:2404540 [wandb_init.py:init():669] starting backend
2024-12-04 16:00:32,420 INFO    MainThread:2404540 [wandb_init.py:init():673] sending inform_init request
2024-12-04 16:00:32,426 INFO    MainThread:2404540 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-12-04 16:00:32,426 INFO    MainThread:2404540 [wandb_init.py:init():686] backend started and connected
2024-12-04 16:00:32,438 INFO    MainThread:2404540 [wandb_init.py:init():781] updated telemetry
2024-12-04 16:00:32,440 INFO    MainThread:2404540 [wandb_init.py:init():814] communicating run to backend with 90.0 second timeout
2024-12-04 16:00:32,767 ERROR   MainThread:2404540 [wandb_init.py:init():841] encountered error: failed to upsert bucket: returned error 404 Not Found: {"errors":[{"message":"project not found","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-12-04 16:00:32,767 ERROR   MainThread:2404540 [wandb_init.py:init():1266] error in wandb.init()
Traceback (most recent call last):
  File "/home/woocheol/miniconda3/envs/gfnmapf/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 1256, in init
    return wi.init()
  File "/home/woocheol/miniconda3/envs/gfnmapf/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 847, in init
    raise error
wandb.errors.errors.CommError: failed to upsert bucket: returned error 404 Not Found: {"errors":[{"message":"project not found","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-12-04 16:00:32,776 WARNING MsgRouterThr:2404540 [router.py:message_loop():75] message_loop has been closed
